test_awx_connection.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+"""Test AWX database connection"""
+
+import subprocess
+import sys
+
+try:
+    result = subprocess.run(
```

Comment:
The sql to the DB should be done by classic connection object, not using exec and CLI psql, thats not standard way how to do queries in DB :-) Its for some external scripts, but we dont want to do it in our app.